upi/vsphere/modules/aws/README.md

```diff
@@ -0,0 +1,4 @@
+# VMware Cloud on AWS
+This is an internal terraform module only.  
+Not particularly useful for anything other than installing
+OpenShift on VMware Cloud on AWS
```

Comment:
nit: maybe rephrase to:

> This is an internal Terraform module for installing OpenShift in a VMware Cloud on AWS.

That makes implicitly not for other uses, so we don't have to talk about how not-useful it is going to be for those off-label uses ;)